### PR TITLE
Question 1 Done 

### PR DESCRIPTION
**Pull request by MichaelAgonsi**
I am done with Question 1. Please review, comment and approve to the Assignment Main Branch for later final review.

### CODE REVIEW

File: PyThinkers Assignment/Question 1.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                    print("*", end=" ")
+                else:
+                    print(" ", end=" ")
+            print()
+
+
+def main():
+    # introduces the program
+    print('Rectangle Calculator')
+    print()
```
Comment (ZCZDinn) [inline PyThinkers Assignment/Question 1.py]: remove newline placement here

File: PyThinkers Assignment/Question 1.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                print(f"{'Perimeter:':15} {rect.perimeter():.0f}")
+                print(f"{'Area:':15} {rect.area():.0f} ")
+                rect.draw_box()
+                break
+            except ValueError:  # throws the exception error response
+                print('Please enter an integer!')
+            except Exception:
+                print('An unexpected error occurred!')
+
+        cont = input('Continue ? (y/n): ').lower()
```
Comment (ZCZDinn) [inline PyThinkers Assignment/Question 1.py]: '\nContinue? (y/n): '    for formatting

File: PyThinkers Assignment/Question 1.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    print('Rectangle Calculator')
+    print()
+
+    cont = 'y'
+    # starts the loop for the program
+    while cont.lower() == 'y':
+        while True:
+            try:
+                # takes inputs and handles errors upon entry
+                height = int(input(f"{'Height:':15} "))
```
Comment (ZCZDinn) [inline PyThinkers Assignment/Question 1.py]: Newline placement works better here for looping   i.e.('\nHeight")

**Review by ZCZDinn — CHANGES_REQUESTED** [review]
just minor formatting stuff

**Review by MichaelAgonsi — COMMENTED** [review]
Thanks for this

File: PyThinkers Assignment/Question 1.py (hunk 3 above)
Comment (MichaelAgonsi) [reply PyThinkers Assignment/Question 1.py]: Nice thanks!

**Review by ZCZDinn — APPROVED** [review]
Great work!!!!

**Comment by ZCZDinn** [comment]
Question 3 done and documented. It automatically added to the Pull Request from 5 days ago.   **: )**

**Review by AseelElzamel — APPROVED** [review]

**Pull request merged**